PWGCF/FemtoWorld/Core/FemtoWorldContainer.h

```diff
@@ -133,8 +154,11 @@ class FemtoWorldContainer
   static constexpr int mEventType = eventType;                                        ///< Type of the event (same/mixed, according to femtoWorldContainer::EventType)
   float mMassOne = 0.f;                                                               ///< PDG mass of particle 1
   float mMassTwo = 0.f;                                                               ///< PDG mass of particle 2
+  double PHI_LO;
+  double PHI_HI;
+  double PIT = 6.28318530717958623;
```

Comment:
you can make it `static constexpr` as it is a constant. And you can also put this below line 30, like a global constant for all femtoworld-related

Reply:
```suggestion
  static constexpr double PIT = 6.28318530717958623;
```